internal/builtinchecks/yamls/dangling-deployment.yaml

```diff
@@ -0,0 +1,7 @@
+name: "dangling-deployment"
```

Comment:
(Here and for the template) Rename to `mismatching-selector`, the dangling name feels less appropriate given that the selectors are defined in the same object as the template itself.

Reply:
Done